### PR TITLE
Fix crash when load app in head tag

### PR DESCRIPTION
**Pull request by trananhmanh89**
### Environment Info

- Naive UI version: (eg. 2.11.0)
- Browser Info: ( Chome 91)
- System Info: (Windows 10 21H1)

### Steps to reproduce
- Create a static html site
- Build a naive app with a nbutton
- Include app js to head of html file
- App crash, nothing show

### Solution:
- because app js was included to head, so `document.body` is null. Use `document.createElement('div')` instead.

### CODE REVIEW

**Comment by 07akioni** [comment]
Please add changelog to the PR.

**Comment by trananhmanh89** [comment]
> Please add changelog to the PR.

Please tell me how. I don't know.

**Comment by 07akioni** [comment]
> > Please add changelog to the PR.
> 
> Please tell me how. I don't know.

/CHANGELOG.en-US.md <-- add changelog
/CHANGELOG.zh-CN.md <-- add a changelog placeholder, I've fill it

**Comment by trananhmanh89** [comment]
> > > Please add changelog to the PR.
> > 
> > 
> > Please tell me how. I don't know.
> 
> /CHANGELOG.en-US.md <-- add changelog
> /CHANGELOG.zh-CN.md <-- add a changelog placeholder, I'll fill it

Hi,
A silly question. How can i add more commits to this PR? Or i have to create new PR for changelog?

**Comment by 07akioni** [comment]
> > > > Please add changelog to the PR.
> 
> > > 
> 
> > > 
> 
> > > Please tell me how. I don't know.
> 
> > 
> 
> > /CHANGELOG.en-US.md <-- add changelog
> 
> > /CHANGELOG.zh-CN.md <-- add a changelog placeholder, I'll fill it
> 
> 
> 
> Hi,
> 
> A silly question. How can i add more commits to this PR? Or i have to create new PR for changelog?

just change files and commit to your branch

**Comment by trananhmanh89** [comment]
> just change files and commit to your branch

Done.

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Fixes
 
+- Fix crash when load app contains `n-button` in head tag #68
```
Comment (07akioni) [inline CHANGELOG.en-US.md:7]:
```suggestion
- Fix `n-button` causes crash when it's imported in script inside `head` tag. [#68](https://github.com/TuSimple/naive-ui/pull/68)
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 ### Fixes
 
+- Fix crash when load app contains `n-button` in head tag #68
```
Comment (07akioni) [inline CHANGELOG.zh-CN.md:7]: - 修正 `n-button` 在 `head` 内部的 script 被引入造成崩溃

**Pull request merged**